### PR TITLE
Fix scrollbars everywhere

### PR DESCRIPTION
**Pull request by aliabid94**
In the JSON component we added scrollbars to all .blocks, adding scrollbars everywhere. So if any component had a JSON component, we got scrollbars everywhere. Removed this CSS.

Screenshot 2024-09-06 at 7 28 35 AM

### CODE REVIEW

File: js/json/Index.svelte [hunk 1]
```diff
@@ -72,7 +72,8 @@
 </Block>
 
 <style>
-	:global(.block) {
-		overflow-y: scroll !important;
+	:global(.json) {
```
Comment (aliabid94) [inline js/json/Index.svelte:75]: where is this class? I couldn't find it

**Comment by aliabid94** [comment]
LGTM @hannahblair, can confirm this solves the scrollbar everywhere issue. Can you explain when the JSON component will actually show the scroll bar? couldn't make sense of that from the PR.

**Comment by aliabid94** [comment]
Oh I see what you're trying to do with the heights and scrolling. Looks good! Should also be possible with pure CSS and flexbox, but no need to rewrite.

**Comment by abidlabs** [comment]
Cool I'll go ahead and merge this in @aliabid94 @hannahblair, since it looks like all issues are resolved, and this is a good one to get out

**Comment by kexul** [comment]
When the pip package will be updated? this is quite annoying.

**Pull request merged**